README.md

```diff
@@ -1 +1,197 @@
-# T.B.C
+# DDD Familiarity Assessment
+
+The DDD Familiarity Assessment goal is to provide you with a set of questions to assess the DDD knowledge within your organization.
+
+:warning: DDD Familiarity Assessment __does not__ intend to be a Maturity Model, and it should not be used as such.
+
+Instead, we expect to help organizations identify DDD knowledge gaps and use the results to prepare learnings, training, or similar to fill those gaps.
+
+## Familiarity Assessment Survey
```

Comment:
I like this section. I think a self-assessment is a good idea as long as it's not used as a tool for ranking people and they feel safe enough to answer honestly.